3rd-party-licenses.md

```diff
@@ -6,29 +6,31 @@ Shopsys Framework also uses some third-party components and images
 which are licensed under their own respective licenses.
 
 ## Components used by Shopsys Framework
+These components are installed via `composer` or via `npm`.
```

Comment:
I'm a bit confused about which packages are listed here and which are listed just in "Libraries dynamically referenced..." below. Are all non-transitive dependecies (directly mentioned inside one of our `composer.json` or `package.json` files) here?

Reply:
Maybe a slightly longer description would be useful.

Reply:
ddddDDDDOOOONE